project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Login/loginForm.html.twig

```diff
@@ -38,17 +38,15 @@
             <dt></dt>
             <dd>
                 <label class="form-choice">
-                    <span class="form-choice__input">
-                        {{ form_widget(form.rememberMe) }}
-                    </span>
-                    <span class="form-choice__label">
+                    {{ form_widget(form.rememberMe, { attr: { class: "css-checkbox" } }) }}
+                    <label class="css-checkbox__image form-choice__label" for="{{ form.rememberMe.vars.id }}">
```

Comment:
Label inside label? Is it necessary?

Reply:
In our usage we do not need label in label. So I changed form-line to div element.

Reply:
That's good. Thanks.